manifests/init.pp

```diff
@@ -67,6 +68,18 @@
 
   $powershell_filename = regsubst($url, '^(.*\/)(.+?)(?:\.[^\.]*$|$)$', '\2')
 
+
+  if $proxyAddress {
+    warning("${module_name}: The use of proxyAddress in Download_file[${title}] is deprecated. Use proxy_address instead.")
```

Comment:
Okay, although that will show up only in the master logs. I assume that an agent side warning through `notify` has better visibility. Perhaps do both?